src/app/app-routing.module.ts

```diff
@@ -1,15 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
-import { AppRoutePath } from './enums/app-route-path.enum';
+import { appRoutes } from './app.routes';
 import { authRoutes } from './pages/auth/auth.routes';
 import { coursesRoutes } from './pages/courses/courses.routes';
 
 const routes: Routes = [
   ...authRoutes,
   ...coursesRoutes,
-  { path: '', redirectTo: AppRoutePath.Courses, pathMatch: 'full' },
-  { path: '**', redirectTo: AppRoutePath.Courses },
+  ...appRoutes,
```

Comment:
just for future improvements consider to use these routes within separate modules such as `CoursesModule`, `AuthModule`, etc. and use lazy loading.

Reply:
Thanks for advice, I'll use the snippet with complex routing provided by you